File: wiki-java/src/main/java/dev/freya02/wiki/commands/slash/SlashShutdown.java
package dev.freya02.wiki.commands.slash;

import dev.freya02.wiki.switches.DevCommand;
import io.github.freya022.botcommands.api.commands.annotations.Command;

// --8<-- [start:dev_command_annotated_condition-command-java]
@Command
@DevCommand // Our custom condition, this command will only exist if it passes.
public class SlashShutdown {
    /* */
}
// --8<-- [end:dev_command_annotated_condition-command-java]

File: wiki-kotlin/src/main/kotlin/io/github/freya022/wiki/commands/slash/SlashShutdown.kt
package io.github.freya022.wiki.commands.slash

import io.github.freya022.botcommands.api.commands.annotations.Command
import io.github.freya022.wiki.switches.DevCommand

// --8<-- [start:dev_command_annotated_condition-command-kotlin]
@Command
@DevCommand // Our custom condition, this command will only exist if it passes.
class SlashShutdown {
    /* */
}
// --8<-- [end:dev_command_annotated_condition-command-kotlin]
